### ISSUE TITLE
The Chinese client selection text option is not valid

### ISSUE THREAD

**Issue by aixiu** [shagu/pfUI#574]
The font display cannot be updated. It may be that the `fonts.lua` file was not updated.
The update font is invalid in the game. Always show only `FZXHLJW.TTF` .

```
if GetLocale() == "zhCN" then
  STANDARD_TEXT_FONT = "Fonts\\FZXHLJW.TTF"
elseif GetLocale() == "koKR" then
  STANDARD_TEXT_FONT = "Fonts\\2002.TTF"
else
  STANDARD_TEXT_FONT = "Interface\\AddOns\\pfUI\\fonts\\Myriad-Pro.ttf"
end

function pfUI.environment:UpdateFonts()
  -- force locale based fonts
  if pfUI_config and pfUI_config.global and pfUI_config.global.force_region == "1" then
    if GetLocale() == "zhCN" then
      pfUI_config.global.font_default = "Fonts\\FZXHLJW.TTF"
      pfUI_config.global.font_combat = "Fonts\\FZXHLJW.TTF"
      pfUI_config.global.font_unit = "Fonts\\FZXHLJW.TTF"
    elseif GetLocale() == "koKR" then
      pfUI_config.global.font_default = "Fonts\\2002.TTF"
      pfUI_config.global.font_combat = "Fonts\\2002.TTF"
      pfUI_config.global.font_unit = "Fonts\\2002.TTF"
    end
  end
```
`font_default` and  ` font_combat`  and ` font_unit`   Use the same font `Fonts\\FZXHLJW.TTF`  , I think Blizzard's original intention is that each interface can use different fonts.  pfUI No matter how you set it up Fonts All `Fonts\\FZXHLJW.TTF`

**Comment by shagu** [comment]
Disable "Enable Region Compatible Font" and you should be able to change every single font.

**Comment by aixiu** [comment]
Thank you. The problem is solved. Disable "Enable Region Compatible Font" effective 。

**Comment by aixiu** [comment]
fix #574
